.github/workflows/run-tests.yml

```diff
@@ -79,7 +79,7 @@ jobs:
 
             # Most of these integration tests should not depend on which mode they
             # are run in, so just run them in `local`
-          - java-version: '11'
+          - java-version: '17'
```

Comment:
The new test emits some benign warnings on Java 11, so just run on java 17 instead